llumnix/backends/vllm/migrate_backend.py

```diff
@@ -0,0 +1,257 @@
+from abc import ABC, abstractmethod
+from typing import List
+import torch
+
+import ray
+import ray.util.collective as col
+from vllm.logger import init_logger
+from vllm.worker.cache_engine import CacheEngine
+from llumnix.config import MigrationConfig
+
+logger = init_logger(__name__)
+
+class MigrateBackendBase(ABC):
```

Comment:
It seems that MigrateBackendBase is nothing to do with vLLM, it's better to put it outside of vLLM directory.